extra/fuegogin/context.go

```diff
@@ -0,0 +1,123 @@
+package fuegogin
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/go-fuego/fuego"
+)
+
+type ContextWithBody[B any] struct {
+	ginCtx *gin.Context
+}
+
+type ContextNoBody = ContextWithBody[any]
+
+// Body implements fuego.Ctx.
```

Comment:
Use godoc

```suggestion
// Body implements [fuego.Ctx].
```

 But also, this could be better

```suggestion
// Body satisfies [fuego.Ctx].
```

Body itself doesn't implements the interface, it helps to comply and satisfy it